lib/inspec/objects/attribute.rb

```diff
@@ -93,7 +93,7 @@ def to_hash
     def to_ruby
       res = ["#{ruby_var_identifier} = attribute('#{@name}',{"]
       res.push "  title: '#{title}'," unless title.to_s.empty?
-      res.push "  default: #{default.inspect}," unless default.to_s.empty?
+      res.push "  value: #{value.inspect}," unless value.to_s.empty?
```

Comment:
Do you expect profiles generated by this to work on older InSpecs? If so, you should add an additional line:

```
res.push "  default: #{value.inspect}," unless value.to_s.empty?
```

This way any generated code will contain both a `value:` and a `default:` option; newer InSpecs will use `value:`, and older inspecs will use `default:`

Reply:
Yes it will have to work on older InSpec versions to not be a breaking change. Both older and current InSpec should be good with the extra key being passed, right?

Reply:
Yes.